app/components/createContainer/createContainerController.js

```diff
@@ -265,6 +423,10 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     Container.query({}, function (d) {
       var containers = d;
       $scope.runningContainers = containers;
+      // If we got a template, we prefill fields, have to do it after container query cause we need runningContainers
+      if ($stateParams.from !== '') {
+        loadFromContainerSpec();
+      }
```

Comment:
We also need to initialize the  `$scope.fromContainer` to an empty object here. It will be used to display the access control form with default values.

```javascript
if ($stateParams.from !== '') {
  loadFromContainerSpec();
} else {
  $scope.fromContainer = {};
}
```

See my comment below (https://github.com/portainer/portainer/pull/855/files#r128255363)